app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupMakingSettingsViewHolder.java

```diff
@@ -75,35 +82,64 @@ private void applyNumberOfGroups() {
         numGroups.clearFocus();
     }
 
+    @OnCheckedChanged(R.id.autocomplete)
+    void onAutocompleteSelected(boolean checked) {
+        if (checked) {
```

Comment:
Nesting is one of the easiest ways we can make code less readable, so let's merge this if statement and the one below it with &&, so we only have 1 layer of nesting.